src/App/App.js

```diff
@@ -14,7 +14,8 @@ import NavigationComponent from "../components/NavigationComponent";
 import { actionsLogout } from "../actions";
 import SmokeButton from "../components/SmokeButton";
 import AuthorizationPage from "../pages/AuthorizationPage";
```

Comment:
SmokeButton and Authorization imports are on lines 10 and 11 so 15 and 16 these can be removed.